gapis/service/service.proto

```diff
@@ -420,6 +421,21 @@ message MemoryRange {
   uint64 size = 2;
 }
 
+// UsageHints hints to the server the intended usage of the result of a request.
+// This can be used to improve performance and responsiveness of the RPCs.
+message UsageHints {
```

Comment:
maybe...? should this be an enum instead? If we'd ever want to add something variable, wouldn't it be part of the RenderSettings?